src/BenchmarkDotNet/ConsoleArguments/ConfigParser.cs

```diff
@@ -112,7 +112,7 @@ private static bool Validate(CommandLineOptions options, ILogger logger)
                     logger.WriteLineError($"The provided runtime \"{runtime}\" is invalid. Available options are: {string.Join(", ", Enum.GetNames(typeof(RuntimeMoniker)).Select(name => name.ToLower()))}.");
                     return false;
                 }
-                else if (runtimeMoniker == RuntimeMoniker.Wasm && !(options.AOTCompilerMode == MonoAotCompilerMode.wasm) && (options.WasmMainJs == null || options.WasmMainJs.IsNotNullButDoesNotExist()))
```

Comment:
Since the other part of the PR uses `RuntimeSrcDir` I would expect it to be validaten in this PR?

https://github.com/dotnet/BenchmarkDotNet/blob/5196e6f5854c45b3e62a8c9a0a68e92491e70d57/src/BenchmarkDotNet/ConsoleArguments/ConfigParser.cs#L124-L128